README.md

```diff
@@ -19,7 +28,7 @@ A directory containing the source files for the script, and the man page file.
 All additions and updates to the script are welcome.
 
 ## Packaging the Script
-After you update the manifest updater and test the changes, you can build an RPM-based package for the script so that it can be installed on systems that use *yum* or *dnf*.
+After you update the manifest updater and test the changes, you can build an RPM-based package for the script to be installed on systems that use *yum* or *dnf*.
```

Comment:
s/the manifest updater/Pantheon CMD/

This one's on me - mind correcting this one for us at the same time? :)

Reply:
WIth a pleasure :)